tests_lib/javascript/test_a.js

```diff
@@ -0,0 +1,20 @@
+// Tests for a.fut, the increment function
+
+import assert from 'assert/strict';
+
+// Hack for Running generated ES6 modules from Emscripten with Node
+// https://github.com/emscripten-core/emscripten/issues/11792#issuecomment-877120580
+import {dirname} from "path";
+globalThis.__dirname = dirname(import.meta.url).substring(7);
```

Comment:
The real grease is the substring(7) at the end of this line. Without this (file://) is weirdly added to the front of the `__dirname` making it unable to load the `.wasm` module.